examples/terraform/main.tf

```diff
@@ -188,6 +188,23 @@ resource "aws_nat_gateway" "nat" {
   subnet_id     = var.aws_subnet_ids[0]
 }
 
+resource "aws_rds_cluster_instance" "cluster_instances" {
+  count              = 2
+  identifier         = "aurora-cluster-demo-${count.index}"
+  cluster_identifier = aws_rds_cluster.default.id
+  instance_class     = "db.r4.large"
+  engine             = aws_rds_cluster.default.engine
+  engine_version     = aws_rds_cluster.default.engine_version
+}
+
+resource "aws_rds_cluster" "default" {
+  cluster_identifier = "aurora-cluster-demo"
+  availability_zones = ["us-west-2a", "us-west-2b", "us-west-2c"]
```

Comment:
minor thing but all examples use `us-east...` (us-east is often the first AWS region that gets new resources/features so there should always be prices for resources there too).

Reply:
Hmm great I didn't know that 👌  I'll change it and will check the price hash again to ensure it doesn't change (I've used west in test too)

Reply:
done

Reply:
What happens if we choose `us-east` in `aws` provider, but then provide another region here? Is it really created in another region?

Reply:
good question, I haven't tried it but I guess it overrides it.